stack.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from linked_list import LinkedList
+
+
+class Stack(object):
+    def __init__(self, iterable=None):
+        """Create a new Stack with optional given iterable of values."""
+        if iterable:
```

Comment:
Given that the `LinkedList` you created accepts `None` as a value for the `iterable` argument to it's `__init__` method, is there any reason to check for None here?